pkg/globalmanager/controllers/featureextraction/downstream.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package featureextraction
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/watch"
+
+	sednav1 "github.com/kubeedge/sedna/pkg/apis/sedna/v1alpha1"
+	"github.com/kubeedge/sedna/pkg/globalmanager/runtime"
+)
+
+func (c *Controller) syncToEdge(eventType watch.EventType, obj interface{}) error {
```

Comment:
this downstream module will send CR Object to LocalController for management of edge.
however i don't find this CR management in LocalController.